addon/classes/Row.js

```diff
@@ -33,6 +35,17 @@ export default class Row extends Ember.ObjectProxy.extend({
    * @type {String | Array}
    */
   classNames: null,
+
+  /**
+   * Element ID for the corresponding `{{lt-row}}`.
+   *
+   * @property elementId
+   * @type {String}
+   * @readOnly
+   */
+  elementId: computed(function() {
```

Comment:
Ah thats a good call. Maybe change this to `rowId` and add a small comment as to why we didn't go by `id` (just so I randomly change this later 😛 )